lib/util/constants.dart

```diff
@@ -56,7 +56,7 @@ const COLLECTOR_RIGHTS_MOMA_009_UNSUPERVISED_DOCS =
     "/bitmark-inc/feral-file-docs/master/docs/collector-rights/009-unsupervised/en.md";
 
 const POSTCARD_RIGHTS_DOCS =
-    "https://raw.githubusercontent.com/bitmark-inc/autonomy-apps/main/docs/postcard_collector_rights.md";
+    "https://raw.githubusercontent.com/bitmark-inc/autonomy-apps/moma_terms/docs/postcard_collector_rights.md";
```

Comment:
why not merge the `moma_terms` into `main`? we should only keep the stable version in a single branch.

Reply:
i noted the commit [temporary change], i will reverse it after the PR of github docs merged

Reply:
it's merged